Dockerfile.pai

```diff
@@ -0,0 +1,21 @@
+# Once we have an official pai-based MAX build, this should be a multiarch image
+FROM codait/max-base:powerai-latest
+
+RUN ["/bin/bash", "-c", "cd /opt/anaconda3/bin && source activate base && conda install -y tensorflow" ] 
```

Comment:
Is there a reason you are installing with conda instead of pip? Also could you please specify the version you want installed to avoid future tensorflow releases breaking things?

Reply:
Conda is used by the powerai team for license-related reasons. The powerai-based MAX-Base image uses preinstalled conda channels to grab frameworks.